packages/suite/src/hooks/wallet/useCoinmarketSpend.ts

```diff
@@ -1,300 +0,0 @@
-import { createContext, useContext, useEffect, useState } from 'react';
-import { desktopApi } from '@trezor/suite-desktop-api';
-import { UseCoinmarketSpendProps, SpendContextValues } from 'src/types/wallet/coinmarketSpend';
```

Comment:
Please, delete also this file `src/types/wallet/coinmarketSpend`

Reply:
done